cli/blockchain_info/service.py

```diff
@@ -0,0 +1,40 @@
+"""
+Provide implementation of the blockchain info.
+"""
+from accessify import implements
+
+from cli.blockchain_info.interfaces import BlockchainInfoInterface
+
+
+@implements(BlockchainInfoInterface)
+class BlockchainInfo:
+    """
+    Implements blockchain info.
+    """
+
+    def __init__(self, service):
+        """
+        Constructor.
+
+        Arguments:
+            service: object to interact with Remme core API.
+        """
+        self.service = service
+
+    async def get_batch_by_id(self, batch_id):
+        """
+        Get batch by id.
+        """
+        # TODO: change to get_batch_by_id after corresponding changes in remme-python-library
```

Comment:
Move `TODO` section of messy comments to the issues, please.

Reply:
Done.